proteus/Domain.py

```diff
@@ -59,6 +63,144 @@ def getBoundingBox(self):
             self.L += [xMax-xMin]
 
 
+    def writeGeo(self, fileprefix, group_names=False):
```

Comment:
Cool, so adding this in the base class will work. Does it work for all domain types that we currently have, or do we need to check the actual type (or  override writeGeo in the child classes)?